### DISCUSSION TITLE
Minio pod s

### DISCUSSION THREAD

**Discussion by nishadsalam** [minio/minio#20242] [Q&A]
Hello

Im using minio to store the objects in my kubernetes cluster.   As i have 1 master 3 worker nodes.   I see the minio fails even if one pod goes down.  What is the best approach setting up the minio in the kube environment and also it works even if one of the minio pod does down. As I have three pods are running but it will not work even if one  pod fails to start?

**Comment by marktheunissen** [comment]
Hi @nishadsalam can you paste the logs you see when it goes down?

Also all the configuration you are using, as well as the steps you took to get setup?

**Reply by nishadsalam** [reply]
@marktheunissen Im  using helm chart to install the minio and using mode:distributed in the healm chart other than that there is no changes were made on the helm chart.

As of the storage im using longhorn and the problem  is that if the one pod is  go down  the minio stopped working? Is it  necessary run the 4 replicas of the minio ?

Thanks
